process/Meeting-Etiquette.md

```diff
@@ -1,12 +1,14 @@
+# Meeting Etiquette
+
 ## We Are Prepared
 _If we're not early, we're late._
 
 * We show up to meetings early so we can dial in and start up conference lines before our meeting starts.
 * We create an agenda for meetings we organize.
 * When we share our screen:
- * We close out of, or minimize, any windows and tabs that are not pertinent to the meeting.
- * We close out of any messaging applications (Slack, Mail, etc), and put our OS X or Growl notifications into "Do Not Disturb" mode. Option-Click on the Notification Center icon in the menu bar will enable "Do Not Disturb" mode.
- * We perform any required A/V setup before the meeting starts (Connecting to G2M, Turning on TV, etc).
+  * We close out of, or minimize, any windows and tabs that are not pertinent to the meeting.
+  * We close out of any messaging applications (Slack, Mail, etc), and put our OS X or Growl notifications into "Do Not Disturb" mode. Option-Click on the Notification Center icon in the menu bar will enable "Do Not Disturb" mode.
```

Comment:
I would genericize this really quickly and just say "put our notifications into ..." 

the __Growl__ mention is showing its age :)

Reply:
Good point.  I'll open a new PR with this change.  This PR is focused on formatting.